zerver/management/commands/list_realms.py

```diff
@@ -24,20 +24,35 @@ def add_arguments(self, parser: ArgumentParser) -> None:
     def handle(self, *args: Any, **options: Any) -> None:
         realms = Realm.objects.all()
 
-        outer_format = "%-5s %-40s %-40s"
+        outer_format = "%-5s %-40s %-40s %-80s"
         inner_format = "%-40s %s"
         deactivated = False
 
         if not options["all"]:
-            print(outer_format % ("id", "string_id", "name"))
-            print(outer_format % ("--", "---------", "----"))
+            print(outer_format % ("id", "string_id", "name", "domain"))
+            print(outer_format % ("--", "---------", "----", "------"))
 
             for realm in realms:
+                realm_domain = realm.realmdomain_set.all()
```

Comment:
This algorithm is wrong; the `realm_domain`  table is for restricting a realm to domains, not the domain the realm is hosted on.  We should just be printing `realm.host`, or probably better, `realm.uri`, instead.